tools/mod/go.mod

```diff
@@ -79,7 +79,7 @@ require (
 	golang.org/x/exp/typeparams v0.0.0-20221208152030-732eee02a75a // indirect
 	golang.org/x/mod v0.10.0 // indirect
 	golang.org/x/net v0.11.0 // indirect
-	golang.org/x/sync v0.2.0 // indirect
+	golang.org/x/sync v0.3.0 // indirect
```

Comment:
The [test](https://github.com/etcd-io/etcd/blob/120f1d6d058689d42f9f6c7b0aafddcb623f436f/tests/go.mod#L43) directly depends on `golang.org/x/sync`, please also bump it for the test.

Reply:
Apologies. I missed the tests. This is done now. @ahrtr .